### PR TITLE
feat: add provider option to disable ssl verification

### PR DESCRIPTION
**Pull request by jfroche**
this change require https://github.com/ewilde/go-kibana/pull/20

### CODE REVIEW

**Comment by jfroche** [comment]
You might want to fork and push your change here first: https://github.com/ewilde/go-kibana/commit/0f99f55b0f1dcea729f8985edbc6525e48700261 in vendor/gopkg.in/ory-am/dockertest.v3/dockertest.go

File: kibana/provider.go [hunk 1]
```diff
@@ -124,13 +130,14 @@ func GetEnvVarOrDefaultBool(key string, defaultValue bool) bool {
...
 	var err error
-
+  
 	p.once.Do(func() {
 		config := &kibana.Config{
 			ElasticSearchPath: d.Get("elastic_search_path").(string),
 			KibanaBaseUri:     d.Get("kibana_uri").(string),
 			KibanaType:        kibana.ParseKibanaType(d.Get("kibana_type").(string)),
 			KibanaVersion:     d.Get("kibana_version").(string),
+  		Insecure:          d.Get("kibana_insecure").(bool),
```
Comment (jfroche) [inline kibana/provider.go]: This line set the `Insecure` parameter in the kibana client config

**Comment by jfroche** [comment]
Hello @ewilde, the `Insecure` parameter for the kibana client config is defined at line 140 in `kibana/provider.go` or I am missing something else ?

**Comment by ewilde** [comment]
> Hello @ewilde, the `Insecure` parameter for the kibana client config is defined at line 140 in `kibana/provider.go` or I am missing something else ?

@jfroche no sorry missed that thank you

**Pull request merged**